server/errorHandler.ts

```diff
@@ -11,7 +11,7 @@ export default function createErrorHandler(production: boolean) {
 
     if (createError.isHttpError(err)) {
       // authorization errors from interventions service cause a special error page to be displayed
-      if (err.status === 403 && err.response?.body?.accessErrors) {
+      if (err.status === 403 && (!err.external || err.response?.body?.accessErrors)) {
```

Comment:
reading this in english:

if the status code is 403 and either the error is internal, or the error has a response with 'accessErrors' in the body, show the auth error page.